onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -219,44 +357,45 @@ Status Attention<T>::PrePack(const Tensor& weights, int input_idx, AllocatorPtr
     return Status::OK();
   }
 
+  const auto* weights_data = weights.Data<T>();
   const size_t input_hidden_size = static_cast<size_t>(weights_dims[0]);
   const size_t hidden_size_x3 = static_cast<size_t>(weights_dims[1]);
   const size_t hidden_size = hidden_size_x3 / 3;
```

Comment:
nit: move the definiton of hidden_size_x3 and hidden_size to else branch. They are not used when hidden sizes are passed in attributes.